spec/features/reviews/user_deletes_reviews_spec.rb

```diff
@@ -34,6 +34,23 @@
 
     visit neighborhood_path(neighborhood)
 
-    expect(page).to_not have_content("Delete")
+    expect(page).to_not have_content("Delete Review")
+  end
+
+  scenario "admin can successfully delete review" do
+    neighborhood = FactoryGirl.create(:neighborhood)
+    user = FactoryGirl.create(:user, role: 'admin')
+    review = FactoryGirl.create(:review, neighborhood: neighborhood)
+
+    visit new_user_session_path
+    sign_in_as(user)
+
+    visit neighborhood_path(neighborhood)
+
+    click_on "Delete Review"
+
+    expect(page).to have_content("Review deleted")
+    expect(page).to_not have_css("\#id_#{review.id}", text: review.rating)
```

Comment:
consider more intuitive syntax:

```ruby
within("\#id_#{review.id}") do
  expect(page).to have_content(review.rating)
end
```

Also, semantically, you should never prefix an id value with `id`. It should indicate something else. Perhaps `review_#{review.id}` - remember that your markup should have semantic meaning